### ISSUE TITLE
Demag GUI automatically setting data model number

### ISSUE THREAD

**Issue by moonshoes87** [PmagPy/PmagPy#90]
Currently, Demag GUI sets the data model number based on whether there is a magic_measurements.txt file or a measurements.txt file in the WD.  I'd like to change the logic so that it  can be overridden at the command line level by the -DM switch.  Sometimes it is reasonable for there to be a measurements.txt and a magic_measurements.txt file in the same directory -- for example, that is what will happen if you use Pmag GUI to import measurement files using the 2.5 import scripts and then convert them to 3.  

Basically, I propose:

    if -DM in sys.argv:
        user specifies data_model
    elif 'measurements.txt' is a file:
         data_model == 3
    else:
         data_model == 2

I don't like to go poking in Demag GUI without checking in, does this work for you @Swanson-Hysell & @Caoimhinmg ?

**Comment by Caoimhinmg** [comment]
Logic looked good so I just went ahead and did it just put `demag_gui.py -DM 3` or `demag_gui.py -DM 2` or 2.5 or whatever it doesn't care if you pass floats. Let me know if you've any problems with my implementation.
